src/Morpho.sol

```diff
@@ -472,41 +468,46 @@ contract Morpho is IMorphoStaticTyping {
     /// @dev Accrues interest for the given market `marketParams`.
     /// @dev Assumes that the inputs `marketParams` and `id` match.
     function _accrueInterest(MarketParams memory marketParams, Id id) internal {
-        uint256 elapsed = block.timestamp - market[id].lastUpdate;
-
-        if (elapsed == 0) return;
-
+        // The IRM is always called, useful for stateful IRMs.
```

Comment:
maybe we should why is it useful?

Reply:
I can't find something that fits well.. can you suggest something please, otherwise i'll resolve

Reply:
`// Initialize the IRM's state if it is stateful.`